### ISSUE TITLE
No uranium ore generation in minecraft version 1.19.2 (forge version 43.2.21)

### ISSUE THREAD

**Issue by Gesichtsglatze** [BiggerSeries/BiggerReactors#86]
It seems that there is no ore generation in minecraft version 1.19.2.
I see there was once an issue with minecraft version 1.17.2 where the same problem occurred. Unlike here, I can't see anything in the log file about an error registering the ore.

**Comment by gizmocodes** [comment]
Yeeaaaah, this has been a known issue for at least a year. Basically, the old ore generation system we use broke at some point in the past, and fixing/replacing it was a low priority since most players have other mods that generate uranium. I finally sat down, ripped out the old system, and finally redid our uranium ore implementation.

This is now fixed in 2cc36a4 for 1.20.x, and 5acb144 for 1.19.

We're now just waiting on textures (I did some quick and dirty textures for now), but once that's done we should be able to push an update. I'll leave this open until that update is pushed though.

**Comment by Dreadfull0418** [comment]
may there be any mods for uranium generation in 1.20.1 you know of that works? i cant find a single one. and 2 weeks is a bit long just to make ores spawn is it not? sorry for my rudeness, I truly admire mod creators and programmers.

**Comment by gizmocodes** [comment]
My apologies, I was under the impression Rogue was going to trigger a release and didn't check that it actually happened (got lost in communication). Currently talking with him to get it done, will update and close once we have it done!

**Comment by Dreadfull0418** [comment]
no apologies necessary. you guys are doing great. it cant be helped that it takes time sometimes, sorry about my rudeness earlier again

**Comment by gizmocodes** [comment]
No worries, jars are now pushed. You can download them now off of Github (under the releases tab) for 1.19.2 and 1.20.1, or wait a bit and they should show up on CurseForge once they're approved. Thanks again for reminding/poking me!
